docs/docs/how-to/previews-deploys-hosting/deploying-to-vercel.md

```diff
@@ -19,8 +21,6 @@ You can use Vercel with Gatsby in multiple ways:
 
 If you want to use a Custom Domain with your Vercel deployment, you can **Add** or **Transfer in** your domain via your Vercel [account Domain settings](https://vercel.com/dashboard/domains).
 
-Vercel doesn't support advanced features like [SSR](/docs/how-to/rendering-options/using-server-side-rendering/), [DSG](/docs/how-to/rendering-options/using-deferred-static-generation/), or [Image CDN](/docs/how-to/images-and-media/using-gatsby-plugin-image/#gatsby-cloud-image-cdn). You can get all features and faster builds by signing up to [Gatsby Cloud](/dashboard/signup).
```

Comment:
Vercel doesn't currently support `File` nodes inside of SSR/DSG, as well as the [Image CDN](/docs/how-to/images-and-media/using-gatsby-plugin-image/#gatsby-cloud-image-cdn).